src/styles/blocks/card.scss

```
@@ -0,0 +1,91 @@
+.card {
+  width: $card-width;
+  padding: 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: $basic-margin-top-for-card-elements auto 0;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $first-text-color;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $second-text-color;
+  }
+
+  &__reviews {
+    margin-top: $basic-margin-top-for-card-elements;
+    @extend %display-flex-with-space;
+  }
+
+  &__reviews-paragraph {
+    margin: 0;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $second-text-color;
+  }
+
+  &__price {
+    margin-top: 24px;
+    @extend %display-flex-with-space;
+  }
+
+  &__price-paragraph {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $second-text-color;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $first-text-color ;
+  }
+
+  &__buy-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    margin-top: $basic-margin-top-for-card-elements;
+    width: 100%;
+    height: 40px;
+
+    background-color: $main-color;
+    border: 1px solid $main-color;
+    border-radius: 5px;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: $main-background-color;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    transition: all 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```